packages/next/client/performance-relayer.js

```diff
@@ -1,78 +1,54 @@
-function isTypeSupported(type) {
-  if (self.PerformanceObserver && PerformanceObserver.supportedEntryTypes) {
-    return PerformanceObserver.supportedEntryTypes.includes(type)
-  }
-  return false
-}
+import { getCLS, getFID, getFCP, getLCP, getTTFB } from 'web-vitals'
 
-export function observeLayoutShift(onPerfEntry) {
-  if (isTypeSupported('layout-shift')) {
-    let cumulativeScore = 0
-    const observer = new PerformanceObserver(list => {
-      for (const entry of list.getEntries()) {
-        // Only count layout shifts without recent user input.
-        if (!entry.hadRecentInput) {
-          cumulativeScore += entry.value
-        }
-      }
+export default onPerfEntry => {
+  // Measure Cumulative Layout Shift
+  getCLS(({ id, value }) =>
+    onPerfEntry({
+      name: 'cumulative-layout-shift',
```

Comment:
Suggestion: I wonder if it makes sense to use the acronyms here? I used them in my library—mainly because they're shorter and I wanted to keep the file size down. If this library includes my library but then changes the names, you'll end up with both versions of the strings in the final code.

Reply:
Only reason I didn't use acronyms was because many of the complete string names were previously used by the relayer.

Agreed that it makes sense to keep it the same in case your library changes in the future however, fixed 👍